libraries/AP_Scripting/applets/VTOL-quicktune.lua

```diff
@@ -343,11 +389,37 @@ function get_slew_rate(axis)
    if axis == "YAW" then
       return yaw_srate
    end
+   -- implement a custom slew rate check on the accz axis for jerk cm/s/s/s
+   -- this should be replaced with srate like the other pids
+   if axis == "ACCZ" then
```

Comment:
I think I can use the `slew_rate` on `accz` here instead and remove all this logic below.

Reply:
Fixed.